vaccine_feed_ingest/runners/ca/metrolink/normalize.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
```

Comment:
FYI - you should not need the `noqa` anymore - we've fixed how isort works in CI!